R/aaa.R

```diff
@@ -130,6 +130,7 @@ VALID_SUM_CLASSES <- purrr::map_chr(SUPPORTED_MOD_TYPES,
 
 SUMMARY_DETAILS <- "run_details"
 SUMMARY_HEURISTICS <- "run_heuristics"
+SUMMARY_COND_NUM <- "condition_number"
```

Comment:
eh, maybe but I don't think we need to. That's not an exhaustive list. It's mostly meant to check that we don't accidentally assign the class to something that is the totally wrong type of tibble. I made it non-exhaustive so that it isn't overly brittle. We can discuss further if that doesn't sit right with you.